bitcoin/src/blockdata/block.rs

```diff
@@ -287,9 +287,13 @@ impl Block {
     }
 
     /// Returns the strippedsize of the block.
-    pub fn strippedsize(&self) -> usize {
-        let txs_size: usize = self.txdata.iter().map(Transaction::strippedsize).sum();
-        self.base_size() + txs_size
+    /// #[deprecated(since = "0.31.0", note = "Use Block::stripped_size() instead")]
```

Comment:
Perhaps in comit: ` d2464c19487329549f13edb1485f428ad59615d5` change this to 
```suggestion
    /// #[deprecated(since = "0.31.0", note = "Truncates on 32-bit machines, use Block::stripped_size() instead")]
```